src/Bundle.ts

```diff
@@ -53,6 +53,9 @@ export default class Bundle implements BundleInterface {
   public getIds(): string[] {
     const ids = [];
     for (let i = HEADER_START; i < HEADER_START + 64 * this.length; i += 64) {
+      if (this.binary.subarray(i + 32, i + 64).length === 0) {
```

Comment:
Can you move this into a variable so you don't have to do subarray twice. Yes a micro-optimization hehe

Reply:
Done, thank for suggestion!